src/plugins/wizard/public/application/utils/get_top_nav_config.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: this can be the new license header, right?

Reply:
This code is referenced from a similar function in visualize that has the modified header. You could argue that it is different enough to warrant the new header. But I was told that when in doubt it's better to be safe than sorry